python/neuroglancer/ingest/lib.py

```diff
@@ -322,169 +330,208 @@ def __repr__(self):
 
 class Bbox(object):
 
-    def __init__(self, a, b):
-        self.minpt = Vec3(
-            min(a[0], b[0]),
-            min(a[1], b[1]),
-            min(a[2], b[2])
-        )
+  def __init__(self, a, b):
+    self.minpt = Vec3(
+      min(a[0], b[0]),
+      min(a[1], b[1]),
+      min(a[2], b[2])
+    )
+
+    self.maxpt = Vec3(
+      max(a[0], b[0]),
+      max(a[1], b[1]),
+      max(a[2], b[2])
+    )
+
+  @classmethod
```

Comment:
Bbox is now a kind of hub for converting between different kinds of spatial representations. You can make Bboxes from vectors, parse filenames, generate filenames, create slices, make bboxes from slices and so on.